packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -915,19 +915,31 @@ class IntelliJValidatorOnMac extends IntelliJValidator {
     }
 
     final List<String> split = version.split('.');
-
     if (split.length < 2) {
       return null;
     }
+    final int major = int.parse(split[0]);
```

Comment:
why did you parse these into `int`s?

Reply:
These lines were moved from line 923.  Not the best way to do it, but that's how it was done before.

Reply:
I used it to check the major and minor version numbers. It was simpler to compare these as int since the can be easily converted back to strings.

Reply:
Oh wait @christopherfujino is right, I totally misread this.

Reply:
Right, in the previous version I used an if to check the version numbers. I think the ints can be removed now since they are not used. Can I push an update to this?

Reply:
You can revert them back to `String`s now that you aren't actually checking if it's `> 2020`

Reply:
Done

Reply:
Ahh, again demonstrating the difficulty of reviewing different versions of a PR on github. Thanks for updating, @Nash0x7E2